be/src/storage/rowset/array_column_iterator.cpp

```diff
@@ -15,33 +15,32 @@
 #include "storage/rowset/array_column_iterator.h"
 
 #include "column/array_column.h"
+#include "column/column_access_path.h"
 #include "column/nullable_column.h"
 #include "storage/rowset/scalar_column_iterator.h"
 
 namespace starrocks {
 
 ArrayColumnIterator::ArrayColumnIterator(ColumnReader* reader, std::unique_ptr<ColumnIterator> null_iterator,
                                          std::unique_ptr<ColumnIterator> array_size_iterator,
-                                         std::unique_ptr<ColumnIterator> element_iterator)
+                                         std::unique_ptr<ColumnIterator> element_iterator, ColumnAccessPath* path)
```

Comment:
will path will be modified? prefer using const

Reply:
it's assgin to class's paramter, const is unnecessary?

Reply:
`ColumnAccessPath* const path` to avoid changing the pointer of `path`